pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/ControllerPeriodicTasksIntegrationTests.java

```diff
@@ -386,7 +395,75 @@ public void afterRealtimeSegmentRelocatorTest(ITestContext context) throws Excep
     dropRealtimeTable(relocationTable);
   }
 
-  // TODO: tests for other ControllerPeriodicTasks (RetentionManager, BrokerValidationManager, OfflineSegmentIntervalChecker, RealtimeSegmentValidationManager)
+  @BeforeGroups(groups = "brokerResourceValidationManager")
+  public void beforeBrokerResourceValidationManagerTest(ITestContext context)
+      throws Exception {
+    String table1 = "testTable";
+    context.setAttribute("testTableOne", table1);
+    setupOfflineTable(table1);
+  }
+
+  @Test(groups = "brokerResourceValidationManager")
+  public void testBrokerResourceValidationManager(ITestContext context)
+      throws Exception {
+    // Check that the first table we added doesn't need to be rebuilt(case where ideal state brokers and brokers in broker resource are the same.
+    String table1 = (String) context.getAttribute("testTableOne");
+    String table2 = "testTable2";
+    TableConfig tableConfigOne = new TableConfig.Builder(CommonConstants.Helix.TableType.OFFLINE).setTableName(table1).build();
+    String partitionNameOne = tableConfigOne.getTableName();
+    HelixAdmin helixAdmin = _helixManager.getClusterManagmentTool();
```

Comment:
reuse `_helixAdmin` already present in the base class?

Reply:
Done.